### PR TITLE
[RFR] Add redirect prop to Delete

### PR DESCRIPTION
**Pull request by natrim**
sometimes there is need to redirect to some other location after delete

### CODE REVIEW

**Comment by djhi** [comment]
I don't get it, once deleted, you won't have any *valid* route to the resource. Can you explain a little more ?

**Comment by natrim** [comment]
the redirect prop in AOR accepts not only the `list,show,edit,create` but when you put normal string it gives it directly to redux-router (the default branch in switch in resolveRedirectTo.js ) 

so you can redirect to any url

File: src/mui/delete/Delete.js [hunk 1]
```diff
@@ -56,13 +56,20 @@ class Delete extends Component {
...
         event.preventDefault();
         this.props.crudDelete(
             this.props.resource,
             this.props.id,
             this.props.data,
-            this.getBasePath()
+            this.getBasePath(),
+            this.props.redirect
+                ? this.props.redirect
+                : this.defaultRedirectRoute()
```
Comment (djhi) [inline src/mui/delete/Delete.js]: useless call: can you put `list` directly here or in a const if you prefer

Comment (natrim) [reply src/mui/delete/Delete.js]: well i copied it from `Edit.js` .)

Comment (natrim) [reply src/mui/delete/Delete.js]: but i guess i can just remove the condition directly as `crudDeleteAction` has default 'list'

Comment (djhi) [reply src/mui/delete/Delete.js]: Ah right, well that looks strange to me but anyway, ignore this request

**Comment by djhi** [comment]
Thanks!

**Comment by natrim** [comment]
just wanted to push the removal of `defaultRedirectRoute `, when the github beeped :)

**Pull request merged**